cypress.config.js

```diff
@@ -2,6 +2,8 @@ const { defineConfig } = require("cypress");
 
 module.exports = defineConfig({
   e2e: {
+    "video": false,
+    "screenshotOnRunFailure": false,
```

Comment:
with the new format, the configs should go without quotes:
```
video: false,
screenshotOnRunFailure: false
```

Reply:
Fixed in 5bb9927d79f02245e044f38183a2ba1db0f8ce42